docs/src/PolyhedralGeometry/polyhedral_complexes.md

```diff
@@ -42,6 +42,7 @@ ambient_dim(PC::PolyhedralComplex)
 codim(PC::PolyhedralComplex)
 dim(PC::PolyhedralComplex)
 f_vector(PC::PolyhedralComplex)
+isembedded(PC::PolyhedralComplex)
```

Comment:
Is there a specific reason for this function?

According to your definition in the doc a polyhedral complex is *always* embedded.

Reply:
@YueRen needs complexes that aren't embedded, where we e.g. only know the Hasse diagram. I was going to say that we also have polyhedra that aren't embedded, but they seem to be gone? The definition seems now always to be embedded.